test/unit/component.test.js

```diff
@@ -686,31 +711,31 @@ test('should emit a tap event', function(){
   browser.TOUCH_ENABLED = origTouch;
 });
 
-test('should provide timeout methods that automatically get cleared on component disposal', function() {
-  expect(4);
+QUnit.test('should provide timeout methods that automatically get cleared on component disposal',
```

Comment:
you can put the function back on this line, we are going to work on ignoring the length of unit test name lines